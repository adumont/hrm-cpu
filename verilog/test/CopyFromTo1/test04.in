09 e3
27 c4
30 70
55 fe
18 51
37 36
47 bd
20 3e
24 1b
22 ae
17 90
02 81
09 cd
54 99
47 ee
23 a5
13 6d
19 35
59 6c
22 49
29 b5
25 33
24 cf
41 b1
17 ed
23 f9
58 8f
46 b6
44 12
09 1c
58 8e
29 60
41 1a
56 1c
38 b1
16 28
43 62
17 26
57 1a
34 89
31 35
15 a7
52 30
48 fe
