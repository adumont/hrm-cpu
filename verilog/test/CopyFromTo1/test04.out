e3
c4
70
fe
51
36
bd
3e
1b
ae
90
81
cd
99
ee
a5
6d
35
6c
49
b5
33
cf
b1
ed
f9
8f
b6
12
1c
8e
60
1a
1c
b1
28
62
26
1a
89
